influxdb3_wal/src/object_store.rs

```diff
@@ -905,419 +904,6 @@ mod tests {
     use std::any::Any;
     use tokio::sync::oneshot::Receiver;
 
-    #[tokio::test(flavor = "multi_thread", worker_threads = 2)]
-    async fn write_flush_delete_and_load() {
```

Comment:
why remove this test? If covered elsewhere, probably better to have it gone as it's likely to be brittle given it's down in the weeds of creating the contents manually. Would definitely be better to have a refactored one that uses test helpers that build LP into batches, etc.

Reply:
I didn't intentionally remove this test. In fact I *don't* even remember doing so. I'll put that back in. It should not have been removed